cmd/utils/vmss.go

```diff
@@ -222,6 +226,17 @@ func RunCommand(
 		fmt.Printf("Command: %s\nVirtual Machine Scale Set VM:\n%s\n\n", *command, string(b))
 	}
 
+	s := make(chan os.Signal, 1)
+	signal.Notify(s, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-s
+		logrus.Warn("The requested command hasn't finished yet, hit 'Ctrl+C' again to exit.")
+		logrus.Warn("Please notice the command will continue running in the node anyway, " +
+			"and you will be unable to see the output or run another command until it finishes.")
```

Comment:
nit:
```suggestion
		logrus.Warn("The requested command hasn't finished yet, hit 'Ctrl+C' again to exit anyway.")
		logrus.Warn("However, please notice the command will continue running in the node anyway, " +
			"and you will be unable to see the output or run another command until it finishes.")
```
I'm not sure. I just want to connect that the second phrase snetence is a consequence of hitting Ctrl+C again.